lottie-ios/Classes/Private/LOTAnimationView.m

```diff
@@ -208,14 +210,32 @@ - (CGFloat)_progressForFrame:(NSNumber *)frame {
   if (!_sceneModel) {
     return 0;
   }
-  return ((frame.floatValue - _sceneModel.startFrame.floatValue) / (_sceneModel.endFrame.floatValue - _sceneModel.startFrame.floatValue));
+  CGFloat absoluteProgress = ((frame.floatValue - _sceneModel.startFrame.floatValue) / (_sceneModel.endFrame.floatValue - _sceneModel.startFrame.floatValue));
+  if ([self _isPlayingForwards]) {
+    return absoluteProgress;
+  } else {
+    // If the animation is playing backwards, the progress is inverted.
+    return 1 - absoluteProgress;
```

Comment:
Yes, but speed implicitly changes the animation. 
If you have auto-reverse animation going and you are tracking the animation in realtime via animationProgress it will be unclear where the animation is unless animationProgress is absolute.

Animation progress is intended to be a 1:1 mapping to the entire animation timeline, not to the currently playing animation.

This comes up in a lot of different states.
When `playFromProgress:0.5 toProgress:1` is called then animationProgress is still absolute and reports a from 0.5 to 1 even though the internal animation 0-1 would map 0.5 to 1.

Another way to reverse playing an animation is to call `playFromProgress:1 toProgress:0` with a positive speed. Then when you reverse the speed the animation would actually play forward.